packages/flutter_tools/templates/create/android.tmpl/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
We only need to change this file if we need a new version of the gradle wrapper (not Gradle itself), which should be very rare. The only purpose of the wrapper is to download the right version of Gradle.